.gitignore

```diff
@@ -12,3 +12,6 @@ doc/api/
 # Don't commit pubspec lock file
 # (Library packages only! Remove pattern if developing an application package)
 pubspec.lock
+
+# Avoid IntelliJ adding junk files to the repo
+.idea
```

Comment:
Please, don't. These should be ignored in a user's global git ignore.

The number machine and IDE-specific files could grow forever...